peps/pep-0770.rst

```diff
@@ -13,6 +13,8 @@ Post-History:
   `06-Jan-2025 <https://discuss.python.org/t/76308>`__,
 Resolution: `11-Apr-2025 <https://discuss.python.org/t/76308/112>`__
 
+.. canonical-pypa-spec:: https://packaging.python.org/en/latest/specifications/binary-distribution-format/#the-dist-info-sboms-directory
```

Comment:
This is worse than the original proposal in the PR, we're now tied to the specific location rather than the automatic cross-reference

Reply:
We can update later once @sethmlarson has gotten a `:ref:` at packaging.python.org that can be used.